### PR TITLE
Deadeye

### PR DESCRIPTION
**Pull request by roberthartmann**
cordeye
Files and code lines were added to all known places.

### CODE REVIEW

**Comment by sprunk** [comment]
`cordeadeye`? deye is dragon's eye

**Comment by roberthartmann** [comment]
thx, will do

**Comment by roberthartmann** [comment]
I will have to check if I replaced just the right stuff... but no time today, anymore

File: language/de/units.json [hunk 1]
```diff
@@ -371,6 +371,7 @@
 			"corcv": "Arbeiterfahrzeug",
 			"cordecom": "Kommandant",
 			"cordemon": "Dämon",
+			"cordeadeye": "Präzisionsschütze" ,
```
Comment (hihoman23) [inline language/de/units.json:374]: You only need to add translations in English, the translators will do the rest via Transifex

Comment (roberthartmann) [reply language/de/units.json:374]: KK, I happen to be German, so I thought I could translate at least my own language.

Should I reverse it?

File: language/en/units.json
Comment (hihoman23) [inline language/en/units.json:1]: It also seems like you didn't add a name for the English translation.

File: language/de/units.json (hunk 1 above)
Comment (hihoman23) [reply language/de/units.json:374]: @WatchTheFort What would you say?

Comment (WatchTheFort) [reply language/de/units.json:374]: Better to leave translations to Transifex so they can be reviewed by the translation team

Comment (roberthartmann) [reply language/de/units.json:374]: Will do.

**Pull request merged**